test/stage-options.test.ts

```diff
@@ -53,6 +53,60 @@ describe('github environment', () => {
     });
   });
 
+
+  test('can specify one github environment with url', () => {
+    withTemporaryDirectory((dir) => {
+      const pipeline = new GitHubWorkflow(app, 'Pipeline', {
+        workflowPath: `${dir}/.github/workflows/deploy.yml`,
+        synth: new ShellStep('Build', {
+          installCommands: ['yarn'],
+          commands: ['yarn build'],
+        }),
+      });
+
+      const stage = new Stage(app, 'MyStack', {
+        env: { account: '111111111111', region: 'us-east-1' },
+      });
+
+      new Stack(stage, 'MyStack');
+
+      pipeline.addStageWithGitHubOptions(stage, {
+        gitHubEnvironment: { name: 'test', url: 'test.com' },
+      });
+
+      app.synth();
+
+      expect(readFileSync(pipeline.workflowPath, 'utf-8')).toMatch(/.*environment:\s+name: test\s+url: test\.com.*/m);
+    });
+  });
+
+
+  test('can specify one github environment with url', () => {
```

Comment:
this test looks like a duplicate of the one above @tj-harris

Reply:
They are technically different but I see now that having two is redundant as the multiple environment test below covers the name only specification of a github environment. I will consolidate the tests.

Reply:
Yeah straight up duplicated, removing the extra test.